frontend/src/v5/ui/routes/dashboard/projects/projectSettings/projectSettings.component.tsx

```diff
@@ -172,7 +175,7 @@ export const ProjectSettings = () => {
 						<FormattedMessage id="project.settings.form.successMessage" defaultMessage="The project has been updated successfully." />
 					</SuccessMessage>
 				)}
-				<UnhandledErrorInterceptor expectedErrorValidators={[projectAlreadyExists, isFileFormatUnsupported]} />
+				<UnhandledErrorInterceptor expectedErrorValidators={[projectAlreadyExists, isFileFormatUnsupported, isPathNotFound]} />
```

Comment:
What's the condition to trigger this? As I've changed the URL to use an invalid project id, but nothing happened

Reply:
fetching an non-existing image will trigger a 404 in axios. But we need to fetch it to know it doesnt exist

Reply:
But now, accessing the settings of a project without an image shows the error message like this
![image](https://github.com/user-attachments/assets/60da68dc-baee-43e9-bdb8-412b6cd8cc79)
Should that be the case?